### DISCUSSION TITLE
Require UAC

### DISCUSSION THREAD

**Discussion by samchouse** [tauri-apps/tauri#5409] [Q&A] [answered]
How would I relaunch the app automatically if it was not run as admin and still have `tauri dev` working? Right now I’m doing this with `Powershell Start` but it breaks when the dev server tries to reload because the app is still in use.

**Comment by FabianLars** [comment, marked as answer]
You could add a manifest to your app to require admin permissions. In this case the app wouldn't even start in a non-admin context (so you'll need an admin powershell session for `tauri dev`.
See https://github.com/tauri-apps/tauri/discussions/4201 - you unfortunately have to downgrade tauri-build to 1.0 for this to work.

**Reply by samchouse** [reply]
What's preventing use with `tauri-build` after 1.0?

**Reply by FabianLars** [reply]
The solution i linked above will panic because of this: <https://github.com/tauri-apps/tauri/pull/4840> because you can set the resource file only once.
